java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java

```diff
@@ -2653,7 +2709,6 @@ public ValidationResult isPossibleNumberForTypeWithReason(
   /**
    * Check whether a phone number is a possible number given a number in the form of a string, and
    * the region where the number could be dialed from. It provides a more lenient check than
```

Comment:
A random line was deleted here?

Reply:
Sorry.. Just getting aquanted with VIM (dd command) because of bandwidth issues :)